projects/uitdatabank/models/common-string-uri-image.json

```
@@ -0,0 +1,10 @@
+{
+  "title": "uri",
+  "type": "string",
+  "format": "uri",
+  "pattern": "^http(s?):([/|.|\\w|\\s|-])*\\.(?:jpeg|jpeg|gif|png)",
```

```suggestion
  "pattern": "^http(s?):([/|.|\\w|\\s|-])*\\.(?:jpeg|jpeg|gif|png)$",
```
Otherwise something like this would also be valid: `https://example.com/fake.png/file.pdf`